extensions/adobe/experience/adcloud/searchads/campaign.schema.json

```diff
@@ -0,0 +1,166 @@
+{
+  "meta:license": [
+    "Copyright 2019 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/adcloud/searchads/campaign",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Search Advertising Campaign",
+  "type": "object",
+  "meta:extensible": false,
+  "meta:abstract": false,
+  "meta:auditable": true,
+  "meta:extends": ["https://ns.adobe.com/xdm/data/record"],
+  "description": "Search Advertising Campaigns is a set of ad groups (ads, keywords, and bids) that share a budget, location targeting, and other settings.",
+  "definitions": {
+    "searchads-campaign": {
+      "properties": {
+        "searchads:campaignId": {
+          "title": "Campaign Identifier",
+          "type": "string",
+          "description": "Campaign ID on the search advertising platform."
+        },
+        "searchads:campaignName": {
+          "title": "Campaign Name",
+          "type": "string",
+          "description": "Name of the Campaign."
+        },
+        "searchads:campaignDescription": {
+          "title": "Description",
+          "type": "string",
+          "description": "Description of the Campaign."
+        },
+        "searchads:campaignStatus": {
```

Comment:
A keyword could be disapproved[1], but a campaign isn't [2].

[1] https://support.google.com/google-ads/answer/2453978?hl=en 
[2] https://support.google.com/google-ads/answer/2549115?hl=en

Reply:
(1) - Accounts missing in campaign seems to be a good catch. I missed that earlier. Thanks for catching
(2) - Budget > AUD 150 is not a query criteria for reporting use cases, fields can be concatenated
(3) xdm: searchEngineId has a different semantics that our case, for us both searchEngine and SearchEngineId is actually the advertising platform name (google adwords, MSN adcenter and so on), nothing to do with the search.
(4) Campaign Description is actually a field in user db.